{
    "data_folder_path": "data/CVE-2023-41335",
    "pull_requests_urls": [
        "https://github.com/matrix-org/synapse/pull/16272"
    ],
    "cve_name": "CVE-2023-41335",
    "repo": "matrix-org/synapse",
    "warnings": {
        "cve": "CVE-2023-41335",
        "warnings": [
            {
                "type": "NOT_MODIFIED",
                "file_name": "changelog.d/16272.bugfix",
                "message": "not a modified file (status: FileStatus.ADDED)\n"
            }
        ]
    },
    "files_extension_stats": {
        ".py": 2,
        ".bugfix": 1
    },
    "file_number": 3,
    "removed_files_number": 0,
    "added_files_number": 0,
    "nb_lines_affected": 18,
    "private_key_files": [],
    "languages": [
        "Python"
    ]
}